std/file.d

```diff
@@ -5119,3 +5120,77 @@ string tempDir() @trusted
     myFile.write("hello");
     assert(myFile.readText == "hello");
 }
+
+/**
+* Returns the available space in a file system.
+*
+* On `Windows`, this function uses the result of the Windows API function
+* $(LINK2 https://msdn.microsoft.com/en-us/library/windows/desktop/aa364937(v=vs.85).aspx, $(D GetDiskFreeSpaceExW)).
+*
+* On `POSIX` platforms, it uses the $(LINK2 http://man7.org/linux/man-pages/man3/statvfs.3.html,
+* $(D statvfs)) function.
+*
+* Due to inconsistent behaviour with the `statvfs` `Posix` function,
+* `FreeBSD` does not support `getAvailableDiskSpace`.
+*
+* Params:
+*     path = a valid path to any file in the file system
+*
+* Returns:
+*    ulong which contains the available space in bytes
```

Comment:
Available space in `bytes` (the user already sees the return type and we typically don't mention it in the return type)